src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -115,6 +118,7 @@ export class BaselinesTable extends Component {
                     tableId={ tableId }
                     baselineRowData={ baseline }
```

Comment:
This is definitely out of scope with what you are working on, but I like what you did with passing the `baselineName` in instead of the entire row, so can you update BaselinesTable by passing `baselineId={ baseline[0] }` and update BaselineTableKebab by removing references to `baselineRowData` with `baselineId`?